components/html/modules/head/_css.html

```diff
@@ -5,6 +5,7 @@
 @@if(includeArchivoBlack === false){
 	<link href="https://fonts.googleapis.com/css?family=Archivo+Narrow:400,700" rel="stylesheet"/>
 }
+<link rel="stylesheet" type="text/css" href="https://www.purdue.edu/purdue/fonts/HCo_fonts.css" />
```

Comment:
Could we just separate each individual font face's into separate SCSS files that we include into templates based on use? Would be schveet to reduce the number of requests on pages.